src/sentry/static/sentry/app/views/userFeedback/organizationUserFeedback.jsx

```diff
@@ -1,17 +1,144 @@
 import React from 'react';
+// import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {isEqual} from 'lodash';
+
+import {t} from 'app/locale';
+import withOrganization from 'app/utils/withOrganization';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+import SentryTypes from 'app/sentryTypes';
+import Alert from 'app/components/alert';
+import LoadingError from 'app/components/loadingError';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import CompactIssue from 'app/components/compactIssue';
+import EventUserFeedback from 'app/components/events/userFeedback';
+import space from 'app/styles/space';
 
 import UserFeedbackContainer from './container';
+import {fetchUserFeedback, getQuery} from './utils';
+
+class OrganizationUserFeedback extends React.Component {
```

Comment:
I think we should try to use `AsyncComponent` for these - we get error + loading (even though loading by default will be a bit different than the existing component) handling included.

Reply:
I can tackle this as a follow up - right now just keeping things somewhat similar to the project version